spynnaker/pyNN/models/neuron/synapse_dynamics/synapse_dynamics_stdp.py

```diff
@@ -327,26 +389,23 @@ def read_plastic_synaptic_data(
         connections["delay"][connections["delay"] == 0] = 16
         return connections
 
+    @overrides(AbstractPlasticSynapseDynamics.get_weight_mean)
     def get_weight_mean(self, connector, weights):
-        # pylint: disable=too-many-arguments
-
-        # Because the weights could all be changed to the maximum, the mean
-        # has to be given as the maximum for scaling
-        return self.__weight_dependence.weight_maximum
+        return self.get_weight_maximum(connector, weights)
```

Comment:
Removing the comment here could lead to confusion in the future... !

Reply:
Oops - re-added.